data/hai/hai reveal 6 end.txt

```diff
@@ -317,25 +335,45 @@ mission "Hai Reveal [C05] Cleanup"
 			`	She gives you a hard stare, but her resolve isn't up to the task for more than a couple seconds. She looks away, and you can see her blinking away more tears. "Is this to be my end, then?" she asks. "Is this the price I pay for my efforts?"`
 			label decide
 			choice
-				`	(Spare her.)`
-					goto spare
-				`	(End her.)`
-					goto end
 				`	"Am I to be judge, jury, and executioner?"`
 					goto judge
+					to display
```

Comment:
I would suggest putting these back in because I think the scene is potentially most powerful with no words said. I think the players should have that option.
```suggestion
				`	"Am I to be judge, jury, and executioner?"`
					goto judge
				`	(Spare her silently.)`
					goto spare
				`	(End her wordlessly.)`
					goto end
					to display
```

Reply:
I restored both of the silent options and updated the `to display` logic, which your suggested change broke. There are *way* too many options, so I changed which ones occur before and after asking if you're the judicial system.

Before asking if you're the judicial system, you get the silent options and the "I don't feel like thinking about it" options:

1. Am I to be judge, jury, and executioner?
2. Silent responses
3. This is for all the trouble you've caused me, personally.
4. Movie review responses

After asking if you're the judicial system, you lose the movie review options and gain the "I thought about it" options:

1. Silent responses
2. There is no other home for you now. Let that be your justice.
3. Justice is the price you pay.
4. I've sinned more than you... wrong... galaxy... cinders.
5. This is for all the trouble you've caused me, personally.

Reply:
I like it.